omc certs inspect - panic: runtime error: invalid memory address or nil pointer dereference

Running omc certs inspect crashes with:

$ omc certs inspect

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x148f4d2]

goroutine 1 [running]:
github.com/gmeghnag/omc/cmd/helpers.GetAge({0xc000d96540?, 0xc000122240?}, {{0x4?, 0x238bffd?, 0x3ace400?}})
	/home/runner/work/omc/omc/cmd/helpers/helpers.go:199 +0x32
github.com/gmeghnag/omc/cmd/certs.inspectResources({0xc0005add38?, 0x3?, 0x0?})
	/home/runner/work/omc/omc/cmd/certs/inspect.go:208 +0x34a
github.com/gmeghnag/omc/cmd/certs.glob..func2(0x3aac740?, {0x3affa40?, 0x0?, 0x0?})
	/home/runner/work/omc/omc/cmd/certs/inspect.go:55 +0xb0
github.com/spf13/cobra.(*Command).execute(0x3aac740, {0x3affa40, 0x0, 0x0})
	/home/runner/work/omc/omc/vendor/github.com/spf13/cobra/command.go:920 +0x847
github.com/spf13/cobra.(*Command).ExecuteC(0x3aaef80)
	/home/runner/work/omc/omc/vendor/github.com/spf13/cobra/command.go:1040 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/work/omc/omc/vendor/github.com/spf13/cobra/command.go:968
main.main()
	/home/runner/work/omc/omc/main.go:24 +0x25

--- a/cmd/certs/inspect.go
+++ b/cmd/certs/inspect.go
@@ -205,7 +205,7 @@ func inspectResources(resourceTypes []string) {
 		}
 	}
 	for _, curr := range resources {
-		age := helpers.GetAge(vars.MustGatherRootPath+"/namespaces/"+curr.GetNamespace()+"/core/", curr.GetCreationTimestamp())
+		age := helpers.GetAge(vars.MustGatherRootPath+"/cluster-scoped-resources/certificates.k8s.io/certificatesigningrequests/", curr.GetCreationTimestamp())
 		_list := []string{
 			curr.GetNamespace(),
 			curr.GetName(),

